src/Data/Ini.hs

```diff
@@ -71,15 +71,16 @@ import           Data.Attoparsec.Text
 import           Data.Char
 import           Data.HashMap.Strict        (HashMap)
 import qualified Data.HashMap.Strict        as M
-import           Data.Monoid
+import           Data.Semigroup
```

Comment:
Hi, because of this change I cannot use it now with GHC 7.10.3, it says:

```
Preprocessing library ini-0.3.6...

src/Data/Ini.hs:74:18:
    Could not find module ‘Data.Semigroup’
    It is a member of the hidden package ‘semigroups-0.18.4@semig_L4brxOQXNIS4uabdTOA0y3’.
    Perhaps you need to add ‘semigroups’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
```

This change is not backward compatible?

Reply:
@horejsek this particular one is not (for 7.10), but you can add a PR which does a conditional addition of a `semigroups` package for GHC 7.10 and older, please see an example here https://github.com/informatikr/hedis/blob/master/hedis.cabal#L87